tests/SprykerEcoTest/Zed/UnzerApi/Business/AuthorizableChargeApiCallFacadeTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\UnzerApi\Business;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group UnzerApi
+ * @group Business
+ */
+class AuthorizableChargeApiCallFacadeTest extends UnzerApiFacadeBaseTest
+{
+    /**
+     * @var string
+     */
+    protected const FIXTURE_FILE_NAME = 'chargeResponseBody.json';
+
+    /**
+     * @return void
+     */
+    public function testPerformMarketplaceAuthorizeApiCall(): void
```

Comment:
In case you want to create a separate test file for every facade method, it's better to name the class the same as the Facade method's name. And test methods names should reflect the expected outcome.

Reply:
Same applicable for other test classes and methods in this PR